subsys/partition_manager/pm.yml.trustzone

```diff
@@ -7,6 +7,4 @@ sram_nonsecure:
   region: sram_primary
```

Comment:
Would we get an error if sram_secure had pcd_sram in it's span even though there was no pcd_sram in the build?

If not, then doing this would simplify the build I believe as we could drop the CONFIG_MCUBOOT_IS_PART_OF_TRUSTZONE_BUILD option and the spm.conf file.

Reply:
You are correct, thanks for the suggestion.